src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -642,6 +691,47 @@
     </Touch>
   </Target>
 
+  <!--
+    Add Microsoft.Build.NoTargets and Microsoft.Build.Traversal SDK overrides, for all repos that utilize them,
+    except source-build-reference-packages repo, that builds these packages.
+
+    Also skip if building utility projects.
+  -->
+  <Target Name="AddSdkOverrides"
```

Comment:
Can you please explain how this is connected to NuGet.config src modifications?

Reply:
Sure, I'll update the comment with more details.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18478/commits/a140a6e0b2faa46ff98f6bd69ad66a6c0c5b1bfe